apache/superset
# feat: Self subscribe reports

### SUMMARY
Feature to allow users to create reports from either a chart page or a dashboard. It creates an email-only report with the user as the owner and only recipient. 



### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
![_DEV__snowflake](https://user-images.githubusercontent.com/5186919/127884507-0d6514c4-96b8-42d2-88e4-ef5e190d2307.png)
![USA_Births_Names](https://user-images.githubusercontent.com/5186919/127884687-7366f24d-a197-41a8-b8ce-8ef33a2bcf80.png)


### TESTING INSTRUCTIONS
From a chart or dashboard click on the calendar icon. If you do not have a report, you should be able to create one. If you have a report, you should be able to edit, delete or toggle active status. 

### ADDITIONAL INFORMATION
